webpack.config.js

```diff
@@ -13,12 +13,13 @@ const isTestEnv = env === 'test';
 const mode = isProductionEnv ? 'production' : 'development';
 const hashSuffix = isProductionEnv ? '-[contenthash:8]' : '';
 const devServerPort = process.env.WEBPACK_PORT;
+const devtool = process.env.WEBPACK_DEVTOOL || (isProductionEnv ? false : 'eval-source-map');
```

Comment:
maybe one day we could load this from our configs in s3 or equivalent, so we could deploy these?

Reply:
> maybe one day we could load this from our configs in s3 or equivalent, so we could deploy these?

Could you expand what you had in mind with S3? Technically we can enable these now to be deployed in production, I had simply opted for this route out of build performance's sake (and the difference may not be too much).

Reply:
I've no strong opposition to deploying them everywhere (including production) if we think they'd be helpful. 🤷

Reply:
🤷 I guess our application is open source anyways, and most of the time browsers don't download the sourcemaps right? Only when opening developer tools?

Reply:
Right, any "risk" with the sourcemaps would be exposing the original source files if we wouldn't want to do that. And we already expose them via this repository 😄

Reply:
Ok, I enabled sourcemap output by default in production in 4a29474 and the guidance in 688c564 + c0e9561. It required a bit of additional work since the sourcemaps seemed to use a different content hash than the original source file, so I switched from `contenthash` to `chunkhash` for Webpack output in ed8e011 ([related docs](https://webpack.js.org/configuration/output/#template-strings)).